quiz.css

```diff
@@ -0,0 +1,83 @@
+article {
+	width: 100%
+}
+
+.Iron {
+	width: 300px;

```

Comment:
From the directions:  "Each card should take up 30% of the width of the browser window."